infra/terraform/environments/alpha/main.tf

```diff
@@ -86,6 +86,8 @@ module "container_registry" {
 
 module "data_access" {
     source = "../../modules/data_access"
+
+    region = "${var.region}"
     env = "${var.env}"
     sns_arn = "${var.sns_arn}"
     gh_hook_secret = "${var.gh_hook_secret}"
```

Comment:
I though about that at some point actually, but then I thought it wasn't really needed as it's relatively easy to get from the config.

I'll add an output variable 👍